src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/LanguageServer/LanguageServerHost.cs

```diff
@@ -29,7 +29,7 @@ internal sealed class LanguageServerHost
 
     public LanguageServerHost(Stream inputStream, Stream outputStream, ExportProvider exportProvider, ILogger logger, AbstractTypeRefResolver typeRefResolver)
     {
-        var messageFormatter = CreateJsonMessageFormatter();
+        var messageFormatter = RoslynLanguageServer.CreateJsonMessageFormatter();
```

Comment:
this was the bug - the vscode serializer wasn't adding all of the vs extension converters.  I deleted the vscode one so we always have consistent serialization.